packages/lesswrong/components/tagging/TagPageButtonRow.tsx

```diff
@@ -95,6 +98,22 @@ const styles = (theme: ThemeType) => ({
     ...theme.typography.italic,
   },
   newLensIcon: {},
+  togglePodcastContainer: {
+    alignSelf: 'center',
+    color: theme.palette.text.dim3,
+    height: PODCAST_ICON_SIZE,
+  },
+  audioIcon: {
+    width: PODCAST_ICON_SIZE + (PODCAST_ICON_PADDING * 2) + "px !important",
+    height: PODCAST_ICON_SIZE + (PODCAST_ICON_PADDING * 2) + "px !important",
```

Comment:
Do these actually have to be `!important`?  Surprised if so, given they're just going into a `ForumIcon`.

Reply:
Oh, I think Shoggoth did those and I didn't bother to check if necessary

Reply:
Ok, those are necessary